harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -740,28 +647,95 @@ def _save(self, path: pathlib.Path) -> workload.Response:
 class PyTorchTrial(det.Trial):
     """
     PyTorch trials are created by subclassing the abstract class
-    :class:`PyTorchTrial`.  Users must define all abstract methods to create the
-    deep learning model associated with a specific trial, and to subsequently
-    train and evaluate it.
+    :class:`PyTorchTrial`.
     """
 
+    # TODO(DET-3204): Add the following comments to the docstring.
+    # There are two different methods to define models, optimizers, and LR schedulers.
+    #
+    # 1. Define the abstract methods for the only model, optimizer, and LR scheduler and
+    #    Determine will initialize them for the users. Note this method doesn't support
+    #    multiple models, optimizers, and LR schedulers.
+    #
+    # 2. Initialize models, optimizers, and LR schedulers in the :meth:`__init__`.
+    #    If using this way, users should calculate the gradients with the losses and call an
+    #    optimization step for the optimizers on your own in :meth:`train_batch`.
+    #    Users are able to use arbitrary number of models, optimizers, and LR schedulers
+    #    and run the forward and backward passes and step optimizers in arbitrary orders.
+
     trial_controller_class = PyTorchTrialController
     trial_context_class = PyTorchTrialContext
 
     @abstractmethod
```

Comment:
question: why is the `_init_` an abstract method?

Reply:
It's always an abstract method for all the frameworks. For the new interface, we need users to wrap models, optimizers in `__init__` so it's mandatory.